tasks/UsblDock.cpp

```diff
@@ -85,3 +85,11 @@ void UsblDock::processParticularNotification(NotificationInfo const &notificatio
         return ;
     }
 }
+
+base::samples::RigidBodyState UsblDock::addFrames(base::samples::RigidBodyState const& pose, std::string const& source_frame, std::string const& target_frame)
```

Comment:
Really ... don't create a method to set two attributes, which is used only once. Do it explicitely.